### DISCUSSION TITLE
Is it possible to configure the output of the freetrack 2.0 protocol?

### DISCUSSION THREAD

**Discussion by Ke-nt0** [opentrack/opentrack#1952] [Q&A] [answered]
My problem is the inability to control the camera offsets in War thunder with the axes of my joystick. I use the 3DOF inertia tracker. The offset control settings in the game are blocked when using the Freetrack protocol. 
Is it possible to solve the problem in OpenTrack? Is it possible to configure the freetrack protocol manually?

**Comment by sthalik** [comment]
Maybe the Fusion tracker can help you with it.

**Reply by Ke-nt0** [reply]
As I understand it, a camera is required to use Fusion tracker, but I do not have it

**Comment by Ke-nt0** [comment, marked as answer]
Solved the problem. I chose the Fusion input, selected the input of the sensor and joystick in it and adjusted the axes in it. Surprisingly, I did not find such a simple solution to the problem in various discussions.
